docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,521 @@
+        "strategy": ...
+    },
+
+
+Selection Input
+^^^^^^^^^^^^^^^^
+
+The input can be one of the following:
+
+.. tabs::
+
+    .. tab:: EMBEDDINGS
+
+        The `lightly OSS framework for self supervised learning <https://github.com/lightly-ai/lightly>`_ is used to compute the embeddings.
+        They are a vector of numbers for each sample.
+        
+        You can define embeddings as input using:
+
+        .. code-block:: python
+
+            "input": {
+                "type": "EMBEDDINGS"
+            }
+
+    .. tab:: SCORES
+
+        They are a scalar number for each element. They are **specified by the prediction task and the scorer**:
+
+        .. code-block:: python
+
+            # using your own predictions
+            "input": {
+                "type": "SCORES",
+                "task": "YOUR_TASK_NAME",
+                "score": "uncertainty_entropy"
+            }
+
+            # using the lightly pretagging model
+            "input": {
+                "type": "SCORES",
+                "task": "lightly_pretagging",
+                "score": "uncertainty_entropy"
+            }
+
+        You can specify one of the tasks you specified in your datasource, see :ref:`docker-datasource-predictions` for reference.
+        Alternatively, you can use **lightly_pretagging** as the task to use object detections created by the Lightly Worker itself.
+        See :ref:`docker-pretagging` for reference.
+
+        The supported score types are explained in :ref:`lightly-active-learning-scorers`.
+
+    .. tab:: PREDICTIONS
+
+        .. _worker-selection-predictions:
+
+        The class distribution probability vector of predictions can be used as well. Here, three cases have to be distinguished:
+
+            - **Image Classification** → The probability vector of each sample's prediction is used directly.
+
+            - **Object Detection** → The probability vectors of the class predictions of all objects in an image are summed up.
+
+            - **Object Detection** and using the :ref:`docker-object-level` → Each sample is a cropped object and has a single object prediction, whose probability vector is used.
+
+        This input is **specified using the prediction task**. Furthermore, it should be remembered, which class names are used for this task, as they are needed in later steps.
+        
+        If you use your own predictions (see :ref:`docker-datasource-predictions`), the task name and class names are taken from the specification in the prediction `schema.json`.
+        
+        Alternatively, you can use the Lightly pretagging and the class names are specified here: :ref:`docker-pretagging`. In that case, the task name is `lightly_pretagging`.
+
+
+        .. code-block:: python
+            
+            # using your own predictions
+            "input": {
+                "type": "PREDICTIONS",
+                "task": "my_object_detection_task",
+                "name": "CLASS_DISTRIBUTION"
```

Comment:
Currently, you can only set it to `CLASS_DISTRIBUTION`.
In the future, it will allow to use other parts of the predictions. E.g. we could have "CLASS_DISTRIBUTION_OBJECTNESS_WEIGHTED" which would weight the sum of class distributions of each prediction by the objectless score of this probability. It is just preparation for the future.